src/services/io/podio/JEventProcessorPODIO.cc

```diff
@@ -180,7 +180,6 @@ JEventProcessorPODIO::JEventProcessorPODIO() {
     "ReconstructedChargedParticleAssociations",
     "MCScatteredElectronAssociations",    // Remove if/when used internally
     "MCNonScatteredElectronAssociations", // Remove if/when used internally
-    "ReconstructedChargedParticleIDs",
```

Comment:
Should we add "ReconstructedChargedRealPIDParticleIDs" instead?
ref. 8f66e97c7ad8c8d9f57dc60cbf1332b95d9d73cd

Reply:
there are still ReconstructedChargedRealPIDParticles above, after all

Reply:
Sure, it was hard to determine the motivation for why that was there. If that's the reason and it just wasn't changed, then we should indeed add it.

Reply:
Relations to ParticleIDs would be dangling currently. If we export `ReconstructedChargedRealPIDParticles`, ParticleIDs are probably the most important piece.

The `ParticleIDs` for LUT PID are currently spread across things like `DRICHParticleIDs` and other collections, and that's why `ReconstructedChargedParticleIDs` is currently not a thing.